src/ts/components/adskipbutton.ts

```diff
@@ -32,29 +34,32 @@ export class AdSkipButton extends Button<AdSkipButtonConfig> {
 
     let config = <AdSkipButtonConfig>this.getConfig(); // TODO get rid of generic cast
     let skipMessage = config.skipMessage;
-    // TODO let adEvent = <Events.AdEvent>null;
-    let skipOffset = 0; // TODO adEvent.skipOffset;
+    let skipOffset = -1;
 
     let updateSkipMessageHandler = () => {
       // Display this button only if ad is skippable
-      if (skipOffset) {
+      // non skippable ads will return -1 for skippableAfter
+      if (skipOffset >= 0) {
         this.show();
       } else {
         this.hide();
       }
 
       // Update the skip message on the button
       if (player.getCurrentTime() < skipOffset) {
-        this.setText(
-          StringUtils.replaceAdMessagePlaceholders(skipMessage.countdown, skipOffset, player));
+        this.setText(StringUtils.replaceAdMessagePlaceholders(skipMessage.countdown, skipOffset, player));
```

Comment:
pls specify the exact placeholders in the related interface in https://github.com/bitmovin/player-specification/issues/38#issuecomment-438584897